.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Question
+about: Ask a question about single-spa-angular
+---
+
+### Question
```

Comment:
I find that a lot of people end up using the Question template to get support for their problems. They don't know if it's a bug with single-spa-angular, something else, or with their code, and are just looking for help getting things set up. In those situations, it's useful to ask about their code and environment. This is why in the systemjs template I didn't ask about the environment in the bug report but did ask about their environment in the question template.